api/src/main/java/org/openmrs/module/eptsreports/reporting/library/converter/BirthDateConverter.java

```diff
@@ -0,0 +1,53 @@
+package org.openmrs.module.eptsreports.reporting.library.converter;
+
+import static org.openmrs.module.eptsreports.reporting.utils.EptsReportUtils.formatDate;
+
+import org.openmrs.calculation.result.CalculationResult;
+import org.openmrs.module.reporting.common.Birthdate;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+
+public class BirthDateConverter implements DataConverter {
+
+  private String what;
+
+  public String getWhat() {
+    return what;
+  }
+
+  public void setWhat(String what) {
+    this.what = what;
+  }
+
+  public BirthDateConverter() {}
+
+  public BirthDateConverter(String what) {
+    this.what = what;
+  }
+
+  @Override
+  public Object convert(Object obj) {
+
+    if (obj == null) {
+      return "";
+    }
+    Birthdate birthdate = (Birthdate) obj;
+    if (birthdate.getBirthdate() != null && what.equals("birthDate")) {
```

Comment:
What is the benefit of this approach vs just using SQL and using a date formatting function there?

Reply:
This now done in SQL